### ISSUE TITLE
Course page

### ISSUE THREAD

**Issue by tenhobi** [tenhobi/flashcards#54]
Create course page with description and list of sections below.

Design deppends on #28 and #29.

* [ ] Web
* [ ] Mobile
* [ ] BLoC (?)

**Comment by vojtechdalecky** [comment]
@tenhobi What do you thing about this ?

<img width="416" alt="snimek obrazovky 2018-04-04 v 10 30 03" src="https://user-images.githubusercontent.com/36296407/38296962-4d1cd542-37f3-11e8-928a-21cd293d017b.png">

**Comment by tenhobi** [comment]
I still think this is overthinked and we should kept sections simple, i.e. only with cards and material. Also I think we don't need to show content of each section on course page.

It seems you don't wanna listen those advices, so this is ok by me. We should not spend too much time with wireframes, tho. Let's code. :P

**Comment by vojtechdalecky** [comment]
We won't show the content of each section. It's dropdown menu. IMHO this is better, because we won't switch between too much useless pages (it's confusing for user).

**Comment by Kenny11CZ** [comment]
![image](https://user-images.githubusercontent.com/8071168/38826300-e0dfcb90-419e-11e8-86aa-983efc5bc320.png)
I will implement this, what do you think about it @tenhobi ?

**Comment by tenhobi** [comment]
@Kenny11CZ yep, do it. In the database, there is already `courses` section, so use it for the data storage. Sections should be in it's collection. And each section should have collections Materials and Exercises.

Btw. for now, don't care about the percentage indicator.
